### PR TITLE
chore(grid): update form template documentation and sample

### PR DESCRIPTION
**Pull request by ntacheva**

### CODE REVIEW

File: components/grid/editing/popup.md [hunk 1]
```diff
@@ -210,6 +210,8 @@ The `GridPopupEditFormSettings` nested tag exposes the following parameters to a
 
 @[template](/_contentTemplates/common/popup-edit-customization.md#edit-form-settings)
 
+>important These settings are not applicable if you are using a `<FormTemplate>` with a custom Form component. See more details in [FormTemplate - Specifics]({%slug grid-templates-popup-form%}#specifics).
```
Comment (yordan-mitev) [inline components/grid/editing/popup.md:213]: Don't use the PascalCase form when you are not referring to the template name as used in the code. So, in titles, use "Form Template".
```suggestion
>important These settings are not applicable if you are using a `<FormTemplate>` with a custom Form component. See more details in [Form Template - Specifics]({%slug grid-templates-popup-form%}#specifics).
```

File: components/grid/templates/popup-form-template.md [hunk 2]
```diff
@@ -11,13 +11,28 @@ position: 50
...
+With the `FormTemplate` feature, you can customize the appearance and content of the create/edit Popup window of the Grid. 
 
-You can use the `Context` attribute of the `<FormTemplate>` tag to set the name of the context variable. The context variable is of type `object` and can be cast to the model type to which the Grid is bound.
+>caption In this article:
+* [Basics](#basics)
+* [Specifics](#specifics)
+* [Example](#example)
 
->When using the template, the default Popup form is replaced by the declared content within the `FormTemplate` tag. Consequently, the default `Update` and `Cancel` buttons are removed. This means the [`OnUpdate` and `OnCancel`]({%slug components/grid/editing/overview%}#events) events cannot be triggered. To modify or cancel the update of a record, you need to include custom controls to manage these actions.
+## Basics
```
Comment (yordan-mitev) [inline components/grid/templates/popup-form-template.md:21]: You must love short and generic headings :)
Why not aim for something loaded with more information? For example, ## Using the Popup Form Template. What is the value of "Basics" in terms of SEO?

File: components/grid/templates/popup-form-template.md [hunk 3]
```diff
@@ -11,13 +11,28 @@ position: 50
...
 
->caption Using a `FormTemplate` to modify the Edit/Create Popup window.
+1. Declare the desired custom content inside the `<FormTemplate>` inner tag of the `<GridPopupEditFormSettings>`. For example, [`TelerikForm`]({%slug form-overview%}) or [`EditForm`](https://learn.microsoft.com/en-us/aspnet/core/blazor/forms/?view=aspnetcore-7.0).
+1. The `FormTemplate` provides `context` of type `object`. Cast that to your model, so you can pass it to the custom form.
+1. (optional)Use the `Context` attribute of the `<FormTemplate>` tag to set the name of the context variable.
```
Comment (yordan-mitev) [inline components/grid/templates/popup-form-template.md:25]:
```suggestion
1. (optional) Use the `Context` attribute of the `<FormTemplate>` tag to set the name of the `context` variable.
```

File: components/grid/templates/popup-form-template.md [hunk 4]
```diff
@@ -11,13 +11,28 @@ position: 50
...
+* [Basics](#basics)
+* [Specifics](#specifics)
+* [Example](#example)
 
->When using the template, the default Popup form is replaced by the declared content within the `FormTemplate` tag. Consequently, the default `Update` and `Cancel` buttons are removed. This means the [`OnUpdate` and `OnCancel`]({%slug components/grid/editing/overview%}#events) events cannot be triggered. To modify or cancel the update of a record, you need to include custom controls to manage these actions.
+## Basics
 
->caption Using a `FormTemplate` to modify the Edit/Create Popup window.
+1. Declare the desired custom content inside the `<FormTemplate>` inner tag of the `<GridPopupEditFormSettings>`. For example, [`TelerikForm`]({%slug form-overview%}) or [`EditForm`](https://learn.microsoft.com/en-us/aspnet/core/blazor/forms/?view=aspnetcore-7.0).
+1. The `FormTemplate` provides `context` of type `object`. Cast that to your model, so you can pass it to the custom form.
```
Comment (yordan-mitev) [inline components/grid/templates/popup-form-template.md:24]: Is `context` a property?
```suggestion
1. The `FormTemplate` provides a `context` property of type `object`. To pass that property to the custom form, cast it to your model.
```

File: components/grid/templates/popup-form-template.md [hunk 5]
```diff
@@ -11,13 +11,28 @@ position: 50
...
+1. (optional)Use the `Context` attribute of the `<FormTemplate>` tag to set the name of the context variable.
+
+## Specifics
+
+When using the template, the default Popup form is replaced by the declared content within the `FormTemplate` tag. This introduces the following specifics:
+* The default `Update` and `Cancel` buttons are removed. This means the [`OnUpdate` and `OnCancel`]({%slug components/grid/editing/overview%}#events) events cannot be triggered. To modify or cancel the update of a record, you need to include custom controls to manage these actions.
```
Comment (yordan-mitev) [inline components/grid/templates/popup-form-template.md:30]: UI element labels are [rendered in **bold**](https://docs.telerik.com/style-guide/element-formatting).
```suggestion
* The default **Update** and **Cancel** buttons are removed. This means that the [`OnUpdate` and `OnCancel`]({%slug components/grid/editing/overview%}#events) events cannot be triggered. To modify or cancel the update of a record, you need to include custom controls to manage these actions.
```

File: components/grid/templates/popup-form-template.md [hunk 6]
```diff
@@ -11,13 +11,28 @@ position: 50
...
+* The [`<GridPopupEditFormSettings>` parameters]({%slug components/grid/editing/popup%}#edit-form-customization) do not apply to a custom `TelerikForm` that you may render inside the `<FormTemplate>` tag. Set the desired Form configurations such as `Columns`, `Orientation` and more on the [Form component]({%slug form-overview%}#form-parameters).
```
Comment (yordan-mitev) [inline components/grid/templates/popup-form-template.md:31]:
```suggestion
* The [`<GridPopupEditFormSettings>` parameters]({%slug components/grid/editing/popup%}#edit-form-customization) do not apply to a custom `TelerikForm` that you may render inside the `<FormTemplate>` tag. Set the desired Form configurations such as `Columns`, `Orientation`, and more on the [Form component]({%slug form-overview%}#form-parameters).
```

**Review by dimodi — APPROVED** [review]

File: components/grid/templates/popup-form-template.md (hunk 2 above)
Comment (ntacheva) [inline components/grid/templates/popup-form-template.md:21]:
```suggestion
## Using the Popup Form Template
```

File: components/grid/templates/popup-form-template.md (hunk 4 above)
Comment (ntacheva) [reply components/grid/templates/popup-form-template.md:24]: It's more of an object but I want to avoid repeating that. I will refactor this sentence.

**Pull request merged**